typhon/cloudmask/aster.py

```diff
@@ -594,20 +594,22 @@ def dt_estimate_scanlines(self, sensor="vnir"):
         scanlines = {"vnir": 4200, "swir": 2100, "tir": 700}
 
         return np.linspace(-0.5, 0.5, scanlines[sensor]) * dtdelta + self.datetime
-
-    def sensor_angles(self, sensor="vnir"):
-        """Calculate a sun reflection angle for a given ASTER image depending on
-        the sensor-sun geometry and the sensor settings.
+    
+    def sensor_angles(self, channel='1'):
+        """Calculate sensor zenith and azimuth angles.
+        Angles are derived for each pixel depending on the channel and the
```

Comment:
Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. [PEP257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings)